app/models/review.rb

```diff
@@ -1,9 +1,16 @@
 class Review < ActiveRecord::Base
   belongs_to :user
   belongs_to :vendor
+  has_many :votes
 
   validates_presence_of :user_id
   validates_presence_of :vendor_id
   validates_presence_of :rating
   validates_presence_of :body
+
+  def vote_value
+    votes = self.votes
```

Comment:
You can delete line 12 and have everything work just fine! Referencing `votes` within this method will assume you're calling `<this_object>.votes`